torchao/prototype/low_bit_optim/subclass_fp8.py

```diff
@@ -142,10 +172,11 @@ def _(func, types, args, kwargs):
     return OptimStateFp8(
         func(x.codes, *args[1:], **kwargs),
         func(x.scale, *args[1:], **kwargs),
+        func(x.k, *args[1:], **kwargs) if x.k else None
     )
 
 
 if TORCH_VERSION_AT_LEAST_2_5:
     from torch.serialization import add_safe_globals
 
-    add_safe_globals([OptimStateFp8])
+    add_safe_globals([OptimStateFp8])
```

Comment:
I wonder why there is a diff here. Can you remove it? Perhaps missing new line?

Reply:
I have added and removed a newline, even tried with different spaces around it, but I’m still unsure why this is happening

Reply:
will try with spaces and see if i can remove it.

Reply:
Do you by chance use a windows machine? maybe it's due to different newline character used by windows vs linux/macos. If that's the case, it's fine then.

Reply:
i use macos